asv_bench/benchmarks/period.py

```diff
@@ -137,7 +64,68 @@ def time_asfreq():
         self.per.asfreq('A')
 
 
-class period_standard_indexing(object):
+class PeriodIndexConstructor(object):
+    goal_time = 0.2
+
+    params = ['D']
+    param_names = ['freq']
+
+    def setup(self, freq):
+        self.freq = freq
+        self.rng = date_range('1985', periods=1000)
+        self.rng2 = date_range('1985', periods=1000).to_pydatetime()
+
+    def time_from_date_range(self):
+        PeriodIndex(self.rng, freq=self.freq)
+
+    def time_from_pydatetime(self):
+        PeriodIndex(self.rng2, freq=self.freq)
+
+
+class DataFramePeriodColumn(object):
+    goal_time = 0.2
+
+    def setup_cache(self):
+        rng = pd.period_range(start='1/1/1990', freq='S', periods=20000)
+        df = pd.DataFrame(index=range(len(rng)))
+        return rng, df
+
+    def time_setitem_period_column(self, tup):
+        rng, df = tup
+        df['col'] = rng
+
+
+class PeriodIndexAlgorithms(object):
+    goal_time = 0.2
+
+    def setup(self):
+        data = [Period('2011-01', freq='M'), Period('2011-02', freq='M'),
+                Period('2011-03', freq='M'), Period('2011-04', freq='M')]
+        self.index = PeriodIndex(data, freq='M')
+
+    def time_drop_duplicates(self):
+        self.index.drop_duplicates()
+
+    def time_value_counts(self):
+        self.index.value_counts()
+
+
+class PeriodSeriesAlgorithms(object):
+    goal_time = 0.2
+
+    def setup(self):
+        data = [Period('2011-01', freq='M'), Period('2011-02', freq='M'),
+                Period('2011-03', freq='M'), Period('2011-04', freq='M')]
+        self.series = Series(data * 1000)
+
+    def time_drop_duplicates(self):
+        self.series.drop_duplicates()
+
+    def time_value_counts(self):
+        self.series.value_counts()
+
+
+class PeriodStandardIndexing(object):
```

Comment:
I would just do 'Indexing' ?

Reply:
Name used to be `period_standard_indexing`

Reply:
Yes, and the one in your PR is certainly an improvement (since it follows PEP8), but I think the 'period' and 'standard' are just redundant (unless there is another class with non-standard indexing)